intelmq/bots/parsers/cert_eu/parser_csv.py

```diff
@@ -0,0 +1,76 @@
+"""
+CERT-EU parser
+"""
+from intelmq.lib.bot import ParserBot
+
+
+class CertEUCSVParserBot(ParserBot):
+
+    abuse_to_intelmq = {
+        "backdoor": "backdoor",
+        "blacklist": "blacklist",
+        "botnet drone": "botnet drone",
+        "brute-force": "brute-force",
+        "c&c": "c&c",
+        "compromised server": "compromised",
+        "ddos infrastructure": "ddos",
+        "ddos target": "--",
```

Comment:
IntelMQ will refuse the value `--`. We need to think how to map that value to another eCSIRT/IntelMQ type.